components/lib/autocomplete/AutoCompleteBase.js

```diff
@@ -84,6 +85,10 @@ const styles = `
         padding: 0;
         list-style-type: none;
     }
+
+    .selected {
```

Comment:
Selected is weird should it be `p-selected` or is this what the other component does?

Reply:
I can change it.

Reply:
Changed to p-highlight similar to dropdown component.